### ISSUE TITLE
Identity migration in sample throws SqlException

### ISSUE THREAD

**Issue by Ketrex** [aspnet/Identity#1013]
I cloned the Identity repo after upgrading to the 1.0.0-preview2-1-003155 runtime.

I made one change to the sample project and that was to remove the if/else statement around the  Identity database migration to ensure that it always runs at startup. No other code in the sample has been modified.

Upon startup, the migration throws a SqlException every time with the message `Column 'NormalizedName' in table 'AspNetRoles' is of a type that is invalid for use as a key column in an index.` The database is created and the EFMigrationsHistory table is created but no record is added. None of the Identity tables are added to the database.

**Comment by Ketrex** [comment]
Deleting the included migration and adding a new one appears to fix the problem.

**Comment by divega** [comment]
The initial migration contained in the sample seems to have been created with a very old version. It contains:

``` C#
            migrationBuilder.CreateTable(
                name: "AspNetRoles",
                columns: table => new
                {
                    Id = table.Column<string>(nullable: false),
                    ConcurrencyStamp = table.Column<string>(nullable: true),
                    Name = table.Column<string>(nullable: true),
                    NormalizedName = table.Column<string>(nullable: true)
                },
                constraints: table =>
                {
                    table.PrimaryKey("PK_AspNetRoles", x => x.Id);
                });
```

But it should be:

``` C#
            migrationBuilder.CreateTable(
                name: "AspNetRoles",
                columns: table => new
                {
                    Id = table.Column<string>(nullable: false),
                    ConcurrencyStamp = table.Column<string>(nullable: true),
                    Name = table.Column<string>(maxLength: 256, nullable: true),
                    NormalizedName = table.Column<string>(maxLength: 256, nullable: true)
                },
                constraints: table =>
                {
                    table.PrimaryKey("PK_AspNetRoles", x => x.Id);
                });
```
cc @HaoK

**Comment by HaoK** [comment]
Yeah I noticed this while working on the auth update, its already fixed/updated in that PR, will fix this as part of the update that adds the new 2FA providers in

https://github.com/aspnet/Identity/pull/963

**Comment by divega** [comment]
@HaoK can we move this out of 1.1.0? Doesn't sounds like it would block us from shipping.

**Comment by HaoK** [comment]
Fixed in 5aed9742a4f1a35558f5ca644e981398df7017c1
